test/framework/src/main/java/org/elasticsearch/index/MapperTestUtils.java

```diff
@@ -78,8 +78,9 @@ public static MapperService newMapperService(NamedXContentRegistry xContentRegis
 
     public static void assertConflicts(String mapping1,
                                        String mapping2,
-                                       DocumentMapperParser
-                                           parser, String... conflicts) throws IOException {
+                                       DocumentMapperParser parser,
+                                       MapperService mapperService,
```

Comment:
I think this can be left unchanged as well?

Reply:
thanks for catching it, mapper service is unused.